packages/fast-usdc-contract/test/snapshots/fast-usdc.contract.test.ts.md

```diff
@@ -1663,7 +1663,6 @@ Generated by [AVA](https://avajs.dev).
           ibcResultWatcher_kindHandle: 'Alleged: kind',
           ibcResultWatcher_singleton: 'Alleged: ibcResultWatcher',
         },
-        icaAccountToDetails: 'Alleged: weakMapStore',
```

Comment:
this will appear to no longer be in the zone, but it still is, right?

might it be better to keep calling `zone.weakMapStore('icaAccountToDetails')` but make clear at the call site that it's a tombstone?

Reply:
A tombstone is reasonable.  I'll do that.